CODEOWNERS

```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
+/libs-haskell/      @hurryabit @cocreature
+/daml-assistant/    @associahedron @hurryabit
+/daml-script/       @cocreature @hurryabit
```

Comment:
@aherrmann-da should probably also be a code owner on this if we want to be accurate.